nixos/doc/manual/development/option-declarations.xml

```diff
@@ -99,7 +99,7 @@ xlink:href="https://nixos.org/nixpkgs/manual/#sec-package-naming">
   <para>
    As an example, we will take the case of display managers. There is a central
    display manager module for generic display manager options and a module file
-   per display manager backend (sddm, gdm ...).
+   per display manager backend (slim, sddm, gdm ...).
   </para>
```

Comment:
Putting slim first makes it seem like the first one users should try, whereas slim is a very leftfield alternative